reduct/boolean-reduct/rte-helpers.metta

```diff
@@ -36,19 +36,26 @@
 )
 
 ;a function to check whether an n-ary expression is consistent or not.
```

Comment:
This comment doesn't belong here, move it to just above the `isConsistentExp` function